src/static/js/libs/gamepad.js

```diff
@@ -562,12 +562,12 @@
     if (this.gamepads.length > 0) {
       this._fire(Gamepad.Event.TICK, this.gamepads);
     }
-    //window.requestAnimationFrame(function() {
-    //	self._update();
-    //});
-    setTimeout(function () {
-      self._update();
-    }, 100);
+    window.requestAnimationFrame(function() {
```

Comment:
Just tested this on my computer, at 10x CPU slow down. Here are the results:

## 10 hz update rate
![10hzgamepad](https://cloud.githubusercontent.com/assets/12476669/20190315/c1a8950e-a735-11e6-892d-9c6143b253aa.png)

## 60 hz update (a la requestAnimationFrame)
![60hzgamepad](https://cloud.githubusercontent.com/assets/12476669/20190346/e59bae7e-a735-11e6-878c-7dc44e135451.png)

@BrianAdams based on these numbers, do you have a suggestion moving forward?